src/components/Notifications/NotificationsFeedTabs.styled.ts

```diff
@@ -56,7 +63,7 @@ const NotificationFeedTabsListContainer = styled("div")((
     }
   }
 
-  return defaultStyles
+  return { ...defaultStyles, width: "90vw", height: "70vh" }
```

Comment:
Why not setting the width and height already in the `defaultStyles`?

Reply:
good catch